tests/integration/test_model.py

```diff
@@ -157,7 +156,27 @@ def test_clustering_and_prediction(c, training_df):
     c.sql(
         """
         CREATE MODEL my_model WITH (
-            model_class = 'dask_ml.cluster.KMeans'
+            model_class = 'sklearn.cluster.KMeans'
+        ) AS (
+            SELECT x, y
+            FROM timeseries
+            LIMIT 100
+        )
+    """
+    )
+
+    check_trained_model(c)
+
+
+@pytest.mark.gpu
+def test_gpu_clustering_and_prediction(c, gpu_training_df):
+    cluster = LocalCluster()
+    client = Client(cluster)
```

Comment:
This is creating a regular dask cluster not a gpu one. 
Can we explore using the `client` or `gpu_client` fixture https://github.com/dask-contrib/dask-sql/blob/main/tests/integration/fixtures.py#L325

Reply:
I didn't realize that, thanks! Will update now.